lighthouse-core/audits/multi-check-audit.js

```diff
@@ -11,6 +11,15 @@
 
 const Audit = require('./audit');
 
+/** @typedef {
+  Partial<Record<LH.Artifacts.ManifestValueCheckID, boolean>> &
```

Comment:
> Is there a long-term plan here with the multi-check audits?
> I'm not sure the js typedefs are a readability win in this step.

ah, now that you say that, there's no reason it can't keep living in `audit.d.ts`. I was deleting all the `details` types from there, but there's no reason to consider it a `details` type now, it's just a handy typedef. I'll put it back.

(personally I hate multi-check-audit and hope it dies as a base class someday, but that's more the long-long-term plan :)